### PR TITLE
Outcome pair programming

### PR DESCRIPTION
**Pull request by joemoore**

### CODE REVIEW

**Comment by joemoore** [comment]
Ready to merge

File: content/outcomes/application-development/pair-programming.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+- [Describe the mechanics of the driver/navigator style of pair programming](#describe-the-mechanics-of-the-drivernavigator-style-of-pair-programming)
+
+## Define pair programming and remote pair programming
+Pair programming is a software development technique whereby two programmers work on the same problem, at the same time, on the same computer, with the goal of producing high quality software quickly. 
+
+Regarding “on the same computer,” we do not mean two people hunched over a tiny laptop. When co-located, we recommend setting up two large monitors, two sets of keyboards, two mice, and two sets of anything else the pair needs to work effectively together, such as a laptop docking station. 
+
+If the pair is not co-located, [remote pair programming](https://tanzu.vmware.com/content/blog/5-tips-for-effective-remote-pair-programming-while-working-from-home) is a good alternative. Remote pair programming is pair programming with the help of commonly available collaboration software.
+
+## List at least five potential benefits of pair programming
```
Comment (Maarc) [inline content/outcomes/application-development/pair-programming.md:34]: You listed only four benefits ;-)

File: content/outcomes/application-development/pair-programming.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+### Knowledge sharing benefits
+While pair programming, not only are two people working on the same difficult problem together, they are also constantly teaching one another in the following ways: 
+
+*   New team member onboarding
+*   Junior developers learning from senior developers
+*   Senior developers learning from junior developers — it’s true!
+*   Breaking down knowledge silos
+*   Reduce the [“bus factor” or “lottery factor”](https://en.wikipedia.org/wiki/Bus_factor)
+*   Tips and tricks sharing
+
```
Comment (Maarc) [inline content/outcomes/application-development/pair-programming.md:63]: Maybe we can mention the "no superheroes" and evolution to evolve towards a learning organization.

File: content/outcomes/application-development/pair-programming.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+### Team building benefits
+Pair programming isn’t all about the code — it’s also all about the people who write the code. 
+
+*   Shared ownership of the code and product
+*   Empathy building
+*   Interpersonal skills development
+*   Social interaction 
+
```
Comment (Maarc) [inline content/outcomes/application-development/pair-programming.md:71]: Also "Happier, more motivated employees" ?

File: content/outcomes/application-development/pair-programming.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+*   Define pair programming and remote pair programming
+*   List at least five potential benefits of pair programming
+*   List at least five potential challenges of pair programming
+*   Describe the mechanics of the driver/navigator style of pair programming
+
+## Keep Learning
+1. Blog: [What’s the Best Way to Pair?](https://tanzu.vmware.com/content/blog/what-s-the-best-way-to-pair) by Maya Rosecrance and Sarah Connor.
+1. Blog: [5 Tips for Effective Remote Pair Programming While Working from Home](https://tanzu.vmware.com/content/blog/5-tips-for-effective-remote-pair-programming-while-working-from-home)
+1. Video: [I've Pair Programmed for 30,000 Hours: Ask Me Anything! — Joe Moore](https://www.youtube.com/watch?v=RCDfBioUgts)
+
```
Comment (Maarc) [inline content/outcomes/application-development/pair-programming.md:125]: Maybe you could add a reference here to the great publicly available "Parrit" pair roation management tool our colleagues have built: https://parrit.io/

File: content/outcomes/application-development/pair-programming.md (hunk 1 above)
Comment (joemoore) [reply content/outcomes/application-development/pair-programming.md:34]: Perhaps my title is confusing. Though there are 4 sub-sections, I list 18 total benefits.

File: content/outcomes/application-development/pair-programming.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+### Knowledge sharing benefits
+While pair programming, not only are two people working on the same difficult problem together, they are also constantly teaching one another in the following ways: 
+
+*   New team member onboarding
+*   Junior developers learning from senior developers
+*   Senior developers learning from junior developers — it’s true!
+*   Breaking down knowledge silos
+*   Reduce the [“bus factor” or “lottery factor”](https://en.wikipedia.org/wiki/Bus_factor)
+*   Tips and tricks sharing
```
Comment (joemoore) [inline content/outcomes/application-development/pair-programming.md:62]:
```suggestion
*   New team member onboarding
*   Junior developers learning from senior developers
*   Senior developers learning from junior developers — it’s true!
*   Breaking down knowledge silos
*   Collective code ownership 
*   Reduce the [“bus factor” or “lottery factor”](https://en.wikipedia.org/wiki/Bus_factor)
*   Tips and tricks sharing
```

File: content/outcomes/application-development/pair-programming.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+
+### Team building benefits
+Pair programming isn’t all about the code — it’s also all about the people who write the code. 
+
+*   Shared ownership of the code and product
+*   Empathy building
+*   Interpersonal skills development
+*   Social interaction
```
Comment (joemoore) [inline content/outcomes/application-development/pair-programming.md:71]:
```suggestion
*   Social interaction 
*   Happier, more motivated employees
```

File: content/outcomes/application-development/pair-programming.md [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+### Knowledge sharing benefits
+While pair programming, not only are two people working on the same difficult problem together, they are also constantly teaching one another in the following ways: 
+
+*   New team member onboarding
+*   Junior developers learning from senior developers
+*   Senior developers learning from junior developers — it’s true!
+*   Breaking down knowledge silos
+*   Collective code ownership 
+*   Reduce the [“bus factor” or “lottery factor”](https://en.wikipedia.org/wiki/Bus_factor)
+*   Tips and tricks sharing
```
Comment (joemoore) [inline content/outcomes/application-development/pair-programming.md:63]:
```suggestion
*   New team member onboarding
*   Junior developers learning from senior developers
*   Senior developers learning from junior developers — it’s true!
*   Breaking down knowledge silos
*   Countering "hero" culture though collective code ownership 
*   Reduce the [“bus factor” or “lottery factor”](https://en.wikipedia.org/wiki/Bus_factor)
*   Tips and tricks sharing
```

File: content/outcomes/application-development/pair-programming.md (hunk 4 above)
Comment (joemoore) [reply content/outcomes/application-development/pair-programming.md:131]: done

File: content/outcomes/application-development/pair-programming.md [hunk 8]
```diff
@@ -0,0 +1,135 @@
+---
+title: Pair Programming
```
Comment (zmb3) [inline content/outcomes/application-development/pair-programming.md:2]: Consider changing the title to something more outcome oriented. Pair programming is not the outcome, but high quality code, "free" knowledge transfer, less time spent on code review, etc. are the outcomes.

Comment (joemoore) [reply content/outcomes/application-development/pair-programming.md:2]: Interesting point. We'll reconsider all of the Modern Application Development articles/outcomes as a group to find a wholistic theme. The other topics have similar non-outcome titles.

**Review by BrianMMcClain — APPROVED** [review]

**Pull request merged**